scripts/mem_usage.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2014-2017, Linaro Limited
+# All rights reserved.
+
+import argparse
+import re
+import subprocess
+import sys
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+                description='Shows the memory usage of a OP-TEE based '
+			'on ELF sections')
```

Comment:
Any particular reason why this file is indented using four spaces instead of the standard tab?
This line is indented using tabs though.

Reply:
Well, some of our scripts use tabs, others use spaces. It just happens that I started from a template that had spaces...
Personally I prefer tabs so I will change.